examples/virtual-networks/private-link-service/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "resource_group_name" {
+  description = "The name of the resource group the Private Link Service is located in."
+  default = ["example-private-link-service"]
+}
+
+variable "location" {
+  description = "The Azure location where all resources in this example should be created."
+  default = ["WestUS"]
```

Comment:
Is there a reason this is an array not a string?

Reply:
same question

Reply:
I have no idea... lol